bot/logger/logger.py

```diff
@@ -0,0 +1,60 @@
+"""Logger Module. Configures the logger for the discord library and the logger for custom code.
+It also provides the decorator for logging command calls.
+"""
+import functools
+import logging
+
+import discord
+
+from bot import constants
+
+# configure logger for bot
```

Comment:
*Configure the logger for the discord bot.

Reply:
#grammar nazi

Reply:
This is not a docstring so command firm and style is up to the author :P

Reply:
![I don't care.](https://media2.giphy.com/media/26mfghTjMmQjabCXS/giphy.gif)